.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "AuthProvider"]
```

Comment:
Could you be able to publish on npm or bower and use it as a dependency? We could have them included in the ```gh-pages``` branch so it can be published.